app/views/partials/transaction-filter-settings.html

```diff
@@ -223,6 +223,12 @@
   </div>
 
   <div class="transactionsFilterRefreshDiv">
-    <button class="btn btn-primary" ng-click="applyFiltersToUrl();"><i class="glyphicon glyphicon-refresh"></i> Apply Filters</button>
-    <button class="btn btn-primary" ng-click="clearFilters();"><i class="glyphicon glyphicon-remove"></i> Clear Filters</button>
+    <div style="float: right; margin-bottom: 5px;">
+      <button class="btn btn-primary" ng-click="applyFiltersToUrl();"><i class="glyphicon glyphicon-refresh"></i> Apply Filters</button>
+      <button class="btn btn-primary" ng-click="clearFilters();"><i class="glyphicon glyphicon-remove"></i> Clear Filters</button>
+    </div>
+
+    <div style="">
+      <button class="btn btn-primary" ng-click="applyFiltersToUrl('bulkrerun');" style="float: right;"><i class="glyphicon glyphicon-repeat"></i> Apply filters for Bulk Rerun ( Find all Transactions )</button>
```

Comment:
Could we change the text here to read 'Re-run all transaction that match current filters'.